app/controllers/pieces_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
   def update
     @piece = Piece.find(params[:id])
     @game = Game.find(@piece.game_id)
-    @piece.update_attributes(piece_params)
+    @piece.move_to!(piece_params[:row_position].to_i, piece_params[:col_position].to_i)
```

Comment:
In that case, I don't really think we need to use strong params here -- we're not accepting any user input that we need to protect against.  You can just directly access the params hash instead.